include/rocksdb/env.h

```diff
@@ -772,6 +772,13 @@ class RandomAccessFile {
   // RandomAccessFileWrapper too.
 };
 
+// A data structure brings the data verification information, which is
+// used togther with data being written to a file.
+struct DataVerificationInfo {
+  // checksum of the data being written.
+  Slice checksum;
+};
+
```

Comment:
Can you just leave this in file_system and make it a forward declaration?

Reply:
Yes. I agree, it will be easier for future deprecation from Env.